garage/tf/algos/reps.py

```diff
@@ -399,19 +414,20 @@ def _dual_opt_input_values(self, samples_data):
     def _policy_opt_input_values(self, samples_data):
         """Update policy optimize input values based on samples data."""
         policy_state_info_list = [
-            samples_data["agent_infos"][k]
+            samples_data['agent_infos'][k]
             for k in self.policy.state_info_keys
         ]   # yapf: disable
         policy_old_dist_info_list = [
-            samples_data["agent_infos"][k]
+            samples_data['agent_infos'][k]
             for k in self.policy.distribution.dist_info_keys
         ]
 
+        # pylint: disable=locally-disabled, unexpected-keyword-arg
```

Comment:
why the unexpected keyword arg?

Reply:
pylint complaints that `obs_var` and `action_var` are invalid arguments. I think it is because `_policy_opt_inputs` are generated function and pylint just can't do a correct static analysis on them.